<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaEkran</class>
 <widget class="QWidget" name="AnaEkran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600000</width>
    <height>1600000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>H2Open ©2024</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page_aramaVeSatis">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>8</horstretch>
           <verstretch>8</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QPushButton" name="pushButton_68">
               <property name="text">
                <string>Satış</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_69">
               <property name="text">
                <string>Araca Bildir</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_70">
               <property name="text">
                <string>Verildi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_71">
               <property name="text">
                <string>Düzenle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_72">
               <property name="text">
                <string>Sil</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_73">
               <property name="text">
                <string>Emanet</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_74">
               <property name="text">
                <string>SMS Gönder</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_75">
               <property name="text">
                <string>Sipariş Fişi</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_76">
               <property name="text">
                <string>Satış Raporu</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_77">
               <property name="text">
                <string>Yazdır</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Minimum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>600</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Tarih:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Müşteri Bilgileri:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Ürün:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Ödeme Yeri:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Sipariş Durumu:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Ödeme Şekli:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="6">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Dispenser:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="7">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Fiş No:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="8">
              <widget class="QLabel" name="label">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDateEdit" name="dateEdit"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="comboBox_4"/>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="comboBox_3"/>
             </item>
             <item row="1" column="4">
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
             <item row="1" column="5">
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item row="1" column="6">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item row="1" column="7">
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="8">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Sıfırla</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ContiguousSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="rowCount">
              <number>10</number>
             </property>
             <property name="columnCount">
              <number>18</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_aboneler_2">
       <property name="text">
        <string>Aboneler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_aramaVeSatis_2">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Arama &amp;&amp; Satış</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>Ürün &amp;&amp; Stok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>Kasa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>Hatırlatmalar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>SMS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>Diğer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>Araçlar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>Yardım</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_kapat_2">
       <property name="text">
        <string>Kapat</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>600</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>on_page_switch_clicked()</slot>
 </slots>
</ui>
